outputs.tf

```diff
@@ -1,14 +1,9 @@
-output "id" {
-  description = "ID of the created example"
-  value       = module.this.enabled ? module.this.id : null
+output "vpn_endpoint_id" {
+  value       = aws_ec2_client_vpn_endpoint.default.id
+  description = "The ID of the Client VPN Endpoint Connection."
 }
 
-output "example" {
-  description = "Example output"
-  value       = module.this.enabled ? local.example : null
-}
-
-output "random" {
-  description = "Stable random number for this example"
-  value       = module.this.enabled ? join("", random_integer.example[*].result) : null
-}
+output "client_configuration" {
+  value       = awsutils_ec2_client_vpn_export_client_config.default.client_configuration
+  description = "VPN Client Configuration data."
+}
```

Comment:
Lets also add

arn - The ARN of the Client VPN endpoint.
dns_name - The DNS name to be used by clients when establishing their VPN session.
id - The ID of the Client VPN endpoint.

Reply:
Maybe add an optional DNS name

https://registry.terraform.io/modules/cloudposse/route53-cluster-hostname/aws/latest